app/views/spree/shared/_upper_footer.html.erb

```diff
@@ -0,0 +1,97 @@
+<div class="footer-links-wrap">
+  <div class="row no-gutters">
+     <div class="col-xl-2 tablet-hide"></div>
+     <div class="col-xl-2 col-lg-4">
+        <div class="footer-logo-wrap">
+           <figure>  <%= image_tag 'footer-logo.svg', alt: "Nature's Flavors" %></figure>
+           <p>Family owned and run for over 40 years. It is our mission to spread the Natural and Organic bounty of nature through our products.</p>
```

Comment:
We need to make this description dynamic as per Spree default

<%# if current_store.description %>
    <div class="d-none d-xl-block pt-xl-2 footer-spree-info">
      <%#= current_store.description %>
    </div>
  <%# end %>
  
  Adjust this code as per template